docs/installation.rst

```diff
@@ -3,104 +3,118 @@
 Installation
 ============
 
-Pre-requisite
--------------
+Welcome to the **ScanCode.io** installation guide! This guide describes how to install ScanCode.io on various platforms.
+Please read and follow the instructions carefully to ensure your installation is functional and smooth.
 
- * **Debian-like** distro or **macOS**
- * **Python 3.6 to 3.9**: https://www.python.org/downloads/
- * Recent version of **git**: https://git-scm.com/
+Supported Platforms
+-------------------
+**ScanCode.io** has been tested and is supported on the following operating systems:
 
-Optional:
- * **PostgreSQL 10** or later: https://www.postgresql.org/ (or https://postgresapp.com/ on macOS)
-
-.. note::
-    ScanCode.io can also be run through **Docker**, this is the preferred approach
-    **on Windows**. Refer to the :ref:`docker_image` chapter for details.
+    #. **Debian-based** Linux distributions
+    #. **MacOS** 10.14 and up
 
+In addition, ScanCode.io can also be run through **Docker**; this is the preferred approach **on Windows**. Refer to the :ref:`docker_image` chapter for details.
 
 .. _system_dependencies:
 
-Local installation
+.. warning::
+    ScanCode.io can **ONLY** be run on Windows through `Docker <https://www.docker.com/>`_ or `Virtual Machines <https://www.virtualbox.org/>`_. However, to avoid any installation issues, it is **NOT RECOMMEND** to run ScanCode.io on Windows machines.
```

Comment:
`**NOT RECOMMEND**` should really be `*Not Recommended*` ? Avoid all caps, use the [inline markups here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup) only, consistently across documentation.